blueprints/ember-cli-blueprint-test-helpers/index.js

```diff
@@ -1,12 +1,62 @@
 var Promise = require('ember-cli/lib/ext/promise');
+var existsSync = require('exists-sync');
+var fs = require('fs-extra');
+var path = require('path');
+var merge = require('lodash/merge');
+var writeFile = Promise.denodeify(fs.outputFile);
 
 module.exports = {
   description: 'Installs dependencies for ember-cli-blueprint-test-helpers',
+  availableOptions: [
+    {
+      name: 'babel',
+      type: Boolean,
+      default: false
+    }
+  ],
   normalizeEntityName: function(){},
-  afterInstall: function() {
-    return Promise.all([
+  afterInstall: function(options) {
+    var afterInstallTasks = [
       this.insertIntoFile('./.npmignore', 'node-tests/'),
-      this.addPackageToProject('mocha', '^2.2.1'),
-    ]);
+      this.addPackageToProject('mocha', '^2.2.1')
+    ];
+    if (options.babel) {
+      afterInstallTasks.splice(0, 0,
+        this.insertIntoFile('./node-tests/mocha.opts', '--require babel-register\n--recursive'),
```

Comment:
does that work? from their documentation it seems that mocha only reads `test/mocha.opts`